templates/Slack/SlackMessageThread.html

```diff
@@ -0,0 +1,78 @@
+<style>
+  .CoveoResultList .coveo-highlight {
+    font-weight: normal;
+    background-color: #FFE300;
+  }
+
+  .CoveoResult {
+    margin-right: 16px;
+  }
+
+  .CoveoImageFieldValue[data-field="@slackmessageauthorprofilepicture"]>span>img {
+    border-radius: 4px;
+  }
+
+  .coveo-folding-footer {
+    order: 1;
+    margin: 0;
+    font-size: 12px;
+  }
+
+  .coveo-folding-results .coveo-result-folding-child-result, .coveo-folding-results .coveo-result-folding-child-result:hover {
+    padding-left: 0;
+    border-left: none;
+  }
+
+  .coveo-folding-oneresult-caption {
+    display: none;
+  }
+</style>
+
+<div class="coveo-result-frame">
+  <div class="coveo-result-row" style="margin-right: 16px;">
+    <div
+      style="font-size: 12px; display: flex; align-items: baseline; margin-bottom: 10px; justify-content: space-between;">
+      <div>
+        <span style="
+          background-color: #86D8FF; color: #FFFFFF; padding: 3px 10px; border-radius: 32px; margin-right: 8px;">
+          THREAD
```

Comment:
Everywhere there is text, since we support localization, we could use the CoveoText component e.g.
```
<div class="CoveoText" data-value="Modified" data-weight="bold"></div>
```